package.json

```diff
@@ -52,14 +52,14 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-router-redux": "^4.0.8",
-    "react-scripts": "3.4.0",
+    "react-scripts": "^3.4.4",
     "react-sortable-tree": "^2.7.1",
     "react-toastify": "^5.5.0",
     "redux": "^4.0.5",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
     "superagent": "^5.2.2",
-    "typescript": "~3.9.5",
+    "typescript": "^3.9.7",
```

Comment:
What was the reasons for updating this package @KyebagonzaJonathanJeremiah

Reply:
I dont remember updating it, so I cant really tell.

Reply:
What do you advise should be done